CIAC 6170 - Azure SQL Management (#26463)
demisto/content: Packs/AzureSQLManagement/ReleaseNotes/1_1_18.md

#### Integrations

##### Azure SQL Management

- Updated the integration and took it out of beta.

> - Added 2 new arguments subscription_id and resource_group_name that if given will override the parameters that were given in the pack configuration for the following commands:

Reviewer: ```suggestion
- Added the subscription_id and resource_group_name arguments to the following commands. If these arguments are provided, they will override the parameters that were provided in the pack configuration.
```

  - azure-sql-db-list
  - azure-sql-servers-list
  - azure-sql-db-audit-policy-list
  - azure-sql-db-audit-policy-create-update
  - azure-sql-db-threat-policy-get
  - azure-sql-db-threat-policy-create-update

> - Updated the configuration parameters 'Subscription ID' and 'Resource Group Name' to be not required, and updated their names to  'Default Subscription ID' and 'Default Resource Group Name'.

Reviewer: ```suggestion
- Updated the configuration parameters 'Subscription ID' and 'Resource Group Name' to be not required, and updated their names to 'Default Subscription ID' and 'Default Resource Group Name'.
```

> - Added new commands:

Reviewer: ```suggestion
- Added the following new commands:
```

#### Integrations

##### Azure SQL Management

> - Updated the integration and took it out of beta.

Reviewer: ```suggestion
Updated the integration and took it out of beta.
```

###### Command: 'azure-sql-db-list'

> - Added 2 new arguments 'subscription_id', 'resource_group_name' that if given will override the parameters that was given in the pack configuration. 

Reviewer 2: Please use the release notes template.
Group all changes together.
For example:
-  Added 2 new arguments *subscription_id* and *resource_group_name* that if given will override the parameters that were given in the pack configuration for the following commands:
  - list the commands

> - Added values to the command's context data.

Reviewer 2: ```suggestion
```
No need

###### Command: 'azure-sql-servers-list'
- Added 2 new arguments 'subscription_id', 'resource_group_name' that if given will override the parameters that was given in the pack configuration.
- The argument 'resource_group_name' can be a list.
- Added an argument 'list_by_resource_group' that if set to 'true' will return the list of servers that matches to the given resource_group_name. 

> - Values were to the command's context data.

Reviewer: ```suggestion
- Values were added to the command's context data.
```
Perhaps you should list the values here?

- The argument 'resource_group_name' can be a list.
- Values were to the command's context data.

###### Command: 'azure-sql-db-audit-policy-create-update'
- Added 2 new arguments 'subscription_id', 'resource_group_name' that if given will override the parameters that was given in the pack configuration.
- The argument 'resource_group_name' can be a list.
- Added an argument 'is_managed_identity_in_use' that if true will add values to the command's context data.

###### Command: 'azure-sql-db-threat-policy-get'
- Added 2 new arguments 'subscription_id', 'resource_group_name' that if given will override the parameters that was given in the pack configuration.

###### Command: 'azure-sql-db-threat-policy-create-update'
- Added 2 new arguments 'subscription_id', 'resource_group_name' that if given will override the parameters that was given in the pack configuration.
- The argument 'resource_group_name' can be a list.

###### New command: 'azure-sql-subscriptions-list'
- Gets all the resource groups for a subscription.

###### New command: 'azure-sql-resource-group-list'

> - Gets all the resource groups for a subscription.

Reviewer 2: - Added new commands:
  - command 1
  - command 2

> 

Reviewer 2: Revert this change
